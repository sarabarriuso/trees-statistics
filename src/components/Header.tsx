import { ReactComponent as EcoLogo } from '../assets/images/ecologi-logo.svg';
import t from '../language';

const Header: React.FC = () => {
  const DEFAULT_CLASSNAME = 'header';
  return (
    <div className={DEFAULT_CLASSNAME}>
      <EcoLogo className={`${DEFAULT_CLASSNAME}__logo`} />
      <h1 className={`${DEFAULT_CLASSNAME}__title`}>
        ðŸŒ³{t.text('app.appHeader')}
      </h1>
      <div className={`${DEFAULT_CLASSNAME}__background`} />
    </div>
  );
};

export default Header;
