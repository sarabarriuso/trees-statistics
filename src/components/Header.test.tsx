import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './Header';

export default describe('Header Tests', () => {
  it('Renders successfully', () => {
    // Arrange
    // Act
    const tree = shallow(<Header />);

    //Assert
    expect(toJson(tree)).toMatchSnapshot();
  });
});
