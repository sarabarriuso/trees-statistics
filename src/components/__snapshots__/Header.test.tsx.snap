// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests Renders successfully 1`] = `
<div
  className="header"
>
  <ForwardRef(SvgEcologiLogo)
    className="header__logo"
  />
  <h1
    className="header__title"
  >
    ðŸŒ³
    Tree Planting Statistics
  </h1>
  <div
    className="header__background"
  />
</div>
`;
